src/Error/ExceptionRenderer.php

```diff
@@ -429,11 +437,20 @@ protected function _outputMessageSafe(string $template): Response
             ->setHelpers([], false)
             ->setLayoutPath('')
             ->setTemplatePath('Error');
-        $view = $this->controller->createView('View');
+
+        try {
+            // Try with app View first with default templates
+            $view = $this->controller->createView();
```

Comment:
This helps applications with custom views like TwigView. They should be allowed to render the default error500 template first.